Open Judge System/OJS.Common/Extensions/FileHelpers.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace OJS.Common.Extensions
+﻿using System.Linq;
```

Comment:
Refactor the using statements to be in the namespace (you can set ReSharper to automatically insert using in the deepest scope)